packages/framework/src/Component/Elasticsearch/ElasticsearchStructureManager.php

```diff
@@ -102,4 +242,104 @@ protected function getDefinition(int $domainId, string $index): array
 
         return $definition;
     }
+
+    /**
+     * @param array $array
+     * @return bool
+     */
+    protected function recursiveArraySort(array &$array)
+    {
+        foreach ($array as &$value) {
+            if (is_array($value)) {
+                $this->recursiveArraySort($value);
+            }
+        }
+        return ksort($array);
+    }
+
+    /**
+     * @param array $currentIndex
+     * @return array
+     */
+    protected function prepareCurrentIndexDefinition(array $currentIndex): array
+    {
+        $currentIndexDefinition = reset($currentIndex);
```

Comment:
please pass into the method `$currentIndexDefinition` because the `$currentIndex` is not used in the function

`prepareCurrentIndexDefinition(reset($currentIndex))`

Reply:
Then you can use just `$indexDefinition` variable name instead of `$currentIndexDefinition`